src/Chapter1.hs

```diff
@@ -556,7 +555,10 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c
+  | c `elem` "aeiou" = True
+  | otherwise = False
```

Comment:
Absolutely correct! And you use eta-reduction 👍🏼 

One note in here, that sometimes, `elem` could be slower than the explicit pattern matching. I remember there were some benchmarks on one particular case, that showed how moving to pattern matching on each case separately drastically decrease time 🐎 

Also, you can write it a bit shorter. Since `elem` already returns `Bool`, you can just:

```haskell
isVowel c = c `elem` "aeiou"
```